src/group_theory/order_of_element.lean

```diff
@@ -656,6 +656,10 @@ end
 let ⟨m, hm⟩ := @order_of_dvd_card_univ _ x _ _ in
 by simp [hm, pow_mul, pow_order_of_eq_one]
 
+@[simp, to_additive] lemma subgroup.pow_index_mem {G : Type*} [group G] (H : subgroup G)
```

Comment:
I'm reluctant about adding this to the default simp-set. It's not a rewriting lemma, and we run a real risk of making the default simp-set very slow by adding all these terminating lemmas to it. `simp` lemmas are indexed by their head symbols. In this case that is `∈`. The more simp-lemmas there are with that head symbol, the slower simp becomes.

Reply:
Oh, sorry. I sort of added `simp` on autopilot since it was on `pow_card_eq_one`, without realizing that `pow_index_mem` used `∈` rather than `=`.